p2p/raintree/network.go

```diff
@@ -151,7 +151,7 @@ func (n *rainTreeNetwork) HandleNetworkData(data []byte) ([]byte, error) {
 		EmitEvent(
 			p2pTelemetry.P2P_EVENT_METRICS_NAMESPACE,
 			p2pTelemetry.RAINTREE_MESSAGE_EVENT_METRIC_NAME,
-			"height", blockHeight,
+			p2pTelemetry.RAINTREE_ATTRIBUTE_NAME_HEIGHT, blockHeight,
```

Comment:
Replace when renamed.

Reply:
<img width="887" alt="Screen Shot 2022-08-08 at 1 55 51 PM" src="https://user-images.githubusercontent.com/1892194/183482607-a4c468b8-da1b-45ad-85c2-24914bd7ba11.png">

I used VSCode's rename symbol feature. Did it for me